### DISCUSSION TITLE
2024-09-09 Meetup @Trever

### DISCUSSION THREAD

**Discussion by DrackThor** [cloud-native-austria/cncf-graz#6] [General]
- [x]  date: 2024-09-09
- [x] host: Trever
- [x] speaker: Roland
- [x] talk: Cloud-Native Observability: Grafana Alloy vs OTEL Collector
- [x] sponsor drinks: Trever
- [x] sponsor food/snacks: Trever
- [x] venue: Trever
- TU - @skillpoint-dev / Roland
- [ ] bevy event - @schadesa 
- [ ] meetup event - @schadesa 
- [ ] linkedin posts - all
- [x] cloud native newsflash - @DrackThor 
- [x] (useful tools - @sadoMasupilami) Postponed October
- [x] pubquiz - @schadesa 
- [ ] SWAG/prices - @DrackThor Liste mit benötigten Items an @skillpoint-dev 
- [ ] SWAG für Innsbruck?

Meetup Agenda:

- Welcome from the Hosts
- RSVP awareness -> Jetbrains Licenses!
- Talk

talk abstract:
```
Cloud-Native Observability: Grafana Alloy vs OTEL Collector:
In this talk, Roland Urbano will dive into the essentials of Cloud-Native observability and explore two leading open-source tools for
 instrumenting your infrastructure, platforms, and applications.

He will start by discussing OpenTelemetry, the emerging standard for metrics, logs, and traces, including how to set up observability pipelines using the OpenTelemetry Collector. 
Following this, he will introduce Grafana Alloy, the successor to Grafana Agent, unveiled at GrafanaCon 2024. Grafana Alloy offers a comprehensive suite of built-in exporters for a wide range of systems and applications, making it a powerful choice for observability. 
Additionally, he will discuss the key differences and benefits of each tool to provide a well-rounded understanding of their capabilities.

```

**Comment by X4mp** [comment]
Date needs to be changed to 09th September

**Comment by DrackThor** [comment]
done
